scanner/src/main/kotlin/experimental/ExperimentalScanner.kt

```diff
@@ -20,8 +20,11 @@
 
 package org.ossreviewtoolkit.scanner.experimental
 
+import java.io.File
```

Comment:
commit(msg): Does this backwards compatibility come with a (negative) performance impact?

If so, what do you think how much effort it would take to implement it in a non-backwards compatible way with better performance?

Reply:
For large repositories with may nested repositories creating the archives might take a while, but this is a temporary solution anyway to make the experimental scanner feature complete to be able to start testing. It will be replaced with the proper implementation later on.